jobs/haproxy/templates/haproxy.ctmpl.erb

```diff
@@ -100,19 +97,3 @@ backend tcp-routers
     <% p("ha_proxy.go_router.servers").each_with_index do |ip, index| %>
         server node<%= index %> <%= ip %>:<%= p("ha_proxy.go_router.port") %> check inter 1000
     <% end %>
-
```

Comment:
The commit message doesn't explain why this is removed. Can you add description?